style.css

```diff
@@ -0,0 +1,537 @@
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  justify-content: space-between;
+  margin-top: 80px;
+}
+
+.mobile-nav-link a {
+  text-decoration: none;
+  color: #fff;
+  font-size: 32px;
+  font-weight: 600;
+}
+
+.live-link,
+.source-link {
+  text-decoration: none;
+}
+
+.nav-link {
+  list-style-type: none;
+  font-size: 15px;
+  font-weight: 500;
+}
+
+.nav-link a {
+  text-decoration: none;
+  color: #344563;
+  padding-bottom: 5px;
+}
+
+.header {
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  color: #172b4d;
+  padding: 128px 24px 0;
+}
+
+/* Headline */
+
+#headline {
+  width: 375px;
+  height: 700px;
+  margin: 0 auto;
+  background: #fff url(./images/header-shapes-mobile.png) 0 45px no-repeat;
+  border-bottom-left-radius: 80px;
+}
+
+.portfolio-description {
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  margin: 12px 24px;
+}
+
+.lets-connect {
+  font-size: 16px;
+  font-weight: 500;
+  color: #7f8cff;
+  margin-left: 24px;
+  margin-bottom: 12px;
+}
+
+.social-media-links {
+  width: 160px;
+  display: flex;
+  justify-content: space-between;
+  margin-left: 24px;
+  padding-left: 0;
+  list-style-type: none;
+}
+
+/* Works Styles */
+
+.works {
+  display: grid;
+  padding: 114px 24px;
+}
+
+.works .work {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background: #fff;
+  margin: 44px 0;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 16px;
+  padding: 12px;
+}
+
+.work ul {
+  display: flex;
+  align-items: center;
+  margin: 20px 0;
+}
+
+.right-block {
+  width: 100%;
+  margin: 12px 0;
+}
+
+.right-block img {
+  width: 100%;
+}
+
+.left-block {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0;
+  width: 100%;
+  margin: 12px 0;
+}
+
+.work.left-block > img {
+  width: 80% !important;
+}
+
+.left-block ul li {
+  margin-right: 12px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.works .work-title {
+  margin: 0;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+.work-stack {
+  list-style-type: none;
+  padding-left: 0;
+}
+
+.work-stack li:nth-child(1) {
+  text-transform: uppercase;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.work-stack li.dot {
+  width: 8px;
+  height: 8px;
+  background: #c1c7d0;
+  border-radius: 30px;
+  margin: 0 10px 0 10px;
+}
+
+.tags {
+  margin: 12px 0;
+  padding-left: 0;
+}
+
+.tags .tag {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  padding: 4px 12px;
+  height: 24px;
+  background: #ebebff;
+  border-radius: 8px;
+  margin: 0 8px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  color: #6070ff;
+  letter-spacing: 0.03em;
+}
+
+.left-block .action button {
```

Comment:
- [ ] Kindly add button interactions (Active, Hover and Pressed) to all the buttons in your design (see design)